<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStructure</class>
 <widget class="QDialog" name="DialogStructure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>600</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NewRecord">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New Record</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewMeal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeleteRecordFloor">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NewRecordDish">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeleteRecordDish">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Record</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Patient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NewRecordPerson">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeleteRecordPatient">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Record</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewPatientMenu">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient/Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewRecordPatientMenu">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteRecordPatientMenu">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmit1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>SubmitAll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveDish">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmitPatientMenu">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>320</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmitPatient">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogStructure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogStructure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
